init/fluent_bit_init_process.go

```diff
@@ -0,0 +1,401 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/sirupsen/logrus"
+)
+
+// static paths
+const (
+	s3FileFolderPath       = "/init/fluent-bit-init-s3-files/"
+	mainConfigFile         = "/init/fluent-bit-init.conf"
+	originalMainConfigFile = "/fluent-bit/etc/fluent-bit.conf"
+	invokeFile             = "/init/invoke_fluent_bit.sh"
+)
+
+var (
+	// default Fluent Bit command
+	fluentBitCommand = "exec /fluent-bit/bin/fluent-bit -e /fluent-bit/firehose.so -e /fluent-bit/cloudwatch.so -e /fluent-bit/kinesis.so"
+
+	// global s3 client and flag
+	s3Client      *s3.S3
+	exists3Client bool = false
+
+	// global ecs metadata region
+	metadataReigon string = ""
+)
+
+// HTTPClient interface
```

Comment:
useless comment, could remove

Reply:
actually go lint will require a comment for structures... preferably more descriptive than this but this is the minimum needed to pass the linter

Reply:
For these interfaces, I think they are useless comment. And maybe we can discuss a standard for our team's code comments, although this is not a very important thing for this moment..